indico/htdocs/sass/partials/_boxes.scss

```diff
@@ -401,16 +397,48 @@ $i-box-padding: 10px;
     padding: 0;
     margin: 0 -10px;
 
-    /* Sortable <li> will use absolute positioning while being moved thus the
-     * <ul> must use relative positioning for correct alignment. */
+    // Sortable <li> will use absolute positioning while being moved thus the
+    // <ul> must use relative positioning for correct alignment.
     position: relative;
 
     > li {
         @include border-top();
-        display: block;
-        padding: 10px 10px;
+        padding: 10px;
         position: relative;
         overflow: hidden;
+
+        // Sortable row with handle on the left
+        &.ui-sortable {
+            @extend .flexrow;
```

Comment:
Sorry I didn't notice it before. When implementing the flexbox framework for Indico, I thought it would be a good idea to extend from the `.flexrow` and `.flexcol` classes. Later I realized that it creates a specificity hell so I stopped using it. It's better to apply `display: flex` instead.